_component_design/search.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Search Bar
+parent: Search and Filters
+layout: component
+category: Components
+usage: >
+  A search box is a combination of input field and submit button. One may think that the search box doesn’t need a design; after all, it’s just two simple elements. But since the search box is one of the most frequently used design element on content-heavy websites, its usability is critical.
```

Comment:
remove leading indent
use articles: of an input field and a submit button
... used design elements

Reply:
updated.